Tests/Pathfinding/testLandmarkData.py

```diff
@@ -0,0 +1,33 @@
+"""
+Created on Aug 11, 2023
+
+@author: CyberiaResurrection
+"""
+
+import unittest
+
+from PyRoute.Calculation.LandmarkData import LandmarkData
+
+
+class MyTestCase(unittest.TestCase):
```

Comment:
Nitpick point: I prefer to name these classes after the file they are in or element they are testings: 
```suggestion
class LandmardDataTest(unittest.TestCase):
```